File: src/main/java/com/chrisrm/idea/themes/literature/club/MTSayoriTheme.java
package com.chrisrm.idea.themes.literature.club;

public final class MTSayoriTheme extends MTDokiDokiTheme {
  public MTSayoriTheme() {
    super("mt.sayori", "Material Sayori", false, "Sayori");
  }

  @Override
  public String getBackgroundColorString() {
    return "f4fbfe";
  }

  @Override
  public String getSecondaryBackgroundColorString() {
    return "c7f2ff";
  }

  @Override
  public String getSecondaryForegroundColorString() {
    return "256fe2";
  }

  @Override
  public String getSelectionForegroundColorString() {
    return "256fe2";
  }

  @Override
  public String getSelectionBackgroundColorString() {
    return "99ebf0";
  }

  @Override
  public String getTreeSelectionBackgroundColorString() {
    return "546eec";
  }

  @Override
  public String getMenuBarSelectionForegroundColorString() {
    return "ffffff";
  }

  @Override
  public String getMenuBarSelectionBackgroundColorString() {
    return "de0a22";
  }

  @Override
  public String getNotificationsColorString() {
    return "c3e8ff";
  }

  @Override
  public String getContrastColorString() {
    return "d8f2ff";
  }

  @Override
  public String getEditorTabColorString() {
    return getContrastColorString();
  }
}

File: src/main/java/com/chrisrm/idea/themes/literature/club/MTSayoriTheme.kt
package com.chrisrm.idea.themes.literature.club

class MTSayoriTheme : MTDokiDokiTheme("mt.sayori", "Material Sayori", false, "Sayori") {

    override fun getBackgroundColorString(): String = "f4fbfe"

    override fun getClubMember(): String = "sayori.png"

    override fun joyfulClubMember(): String = "sayori_joy.png"

    override fun getSecondaryBackgroundColorString(): String = "c7f2ff"

    override fun getSecondaryForegroundColorString(): String = "256fe2"

    override fun getSelectionForegroundColorString(): String = "256fe2"

    override fun getSelectionBackgroundColorString(): String = "99ebf0"

    override fun getTreeSelectionBackgroundColorString(): String = "546eec"

    override fun getMenuBarSelectionForegroundColorString(): String = "ffffff"

    override fun getMenuBarSelectionBackgroundColorString(): String = "de0a22"

    override fun getNotificationsColorString(): String = "c3e8ff"

    override fun getContrastColorString(): String = "d8f2ff"

    override fun getEditorTabColorString(): String = contrastColorString

    override fun getButtonBackgroundColor(): String = contrastColorString

    override fun getButtonForegroundColor(): String = "00559A"
}
